src/views/NewsFeed.vue

```diff
@@ -1103,8 +1130,13 @@ module.exports = {
                 let inbuiltApps = this.getInbuiltApps(file);
                 if (userApps.length == 0) {
                     if (inbuiltApps.length == 1) {
-                        if (inbuiltApps[0].name == 'hex') {
-                            this.openFileOrDir("Drive", path, {filename:""});
+                        if (inbuiltApps[0].name == 'hex' || inbuiltApps[0].name == 'editor') {
```

Comment:
Why is editor here?

Reply:
Does that mean it's going to ask me to install an app every time I open a .java file?